installer/install.iss

```diff
@@ -208,6 +208,9 @@ Type: files; Name: {app}\{#MINGW_BITNESS}\bin\git-*.exe
 Type: files; Name: {app}\{#MINGW_BITNESS}\libexec\git-core\git-*.exe
 Type: files; Name: {app}\{#MINGW_BITNESS}\libexec\git-core\git.exe
 
+; Delete git-lfs wrapper
+Type: files; Name: {app}\cmd\git-lfs.exe
```

Comment:
Does this work also if the file was not created, because `IsComponentSelected('gitlfs')` was false during the installation?

Reply:
Doesn't seem to be a problem.

Reply:
Yes, sorry, those files are deleted gracefully (i.e. if they are already absent, it is not a big deal, I verified that when I added the part where copied .dll files are deleted from `libexec\git-core`).